src/TMap.cpp

```diff
@@ -64,9 +64,12 @@ TMap::TMap(Host* pH)
 , mDefaultVersion(18)
 // maximum version of the map format that this Mudlet can understand and will
 // allow the user to load
-, mMaxVersion(18)
+, mMaxVersion(19)
 // minimum version this instance of Mudlet will allow the user to save maps in
 , mMinVersion(16)
+, mMapSymbolFont(QFont(QStringLiteral("Bitstream Vera Sans Mono"), 12, QFont::Normal))
+, mMapSymbolFontFudgeFactor(1.0)
+, mIsOnlyMapSymbolFontToBeUsed(true)
```

Comment:
What is the idea behind enabling this restriction by default?

Reply:
To match the optimum case - that the font has ALL the glyphs the user wants.  The use of any font represents a fall-back case that can provide more symbols but they need not come from the specified font - and A) might not look so good overall and B) might not look the same as the font used in an external "CharacterMap" utility - for instance `gucharmap` has an option "only show glyphs from this font" - so that the character selected and pasted into the entry widget on the 2D Mapper will look the **same** as is used on the map {the fact that it may not on that entry widget is a niggle that I will try and address but it can be deferred IMHO}.

Reply:
This option being on or off by default is a choice between "working for most people out of the box" and "working for less people but for some the way the map author intended". Right now it's the latter - I had this experience and someone else who tested on Discord did as well. I think it should be off by default, so at least it _works_, and then people can see about how to get it more working _right_. It'll make people happier as opposed it not working at all!